oasis-webapp/src/main/resources/templates/Login.soy

```diff
@@ -173,10 +175,9 @@ aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-d
   <aside class=two-cols>
     <span style="height: 0px;"></span>
     <span>
-      {let $overview kind="text"}{msg desc="overview URL"}https://www.ozwillo.com/en/oz/discover{/msg}{/let}
+      {let $overview kind="text"}{msg desc="overview URL"}https://www.ozwillo.com/en/discover{/msg}{/let}
```

Comment:
[“Cool URIs don't change”](http://www.w3.org/Provider/Style/URI.html), too bad you changed it.
But as said above, this is localizable content so it can't change that way.
I could update all URIs, but unfortunately the new website only has French and English…